commands/command_smudge.go

```diff
@@ -20,6 +22,67 @@ var (
 	smudgeSkip = false
 )
 
+// delayedSmudge performs a 'delayed' smudge, adding the LFS pointer to the
+// `*tq.TransferQueue` "q" if the file is not present locally, passes the given
+// filepathfilter, and is not skipped. If the pointer is malformed, or already
+// exists, it streams the contents to be written into the working copy to "to".
+//
+// delayedSmudge returns the number of bytes written, whether the checkout was
+// delayed, the *lfs.Pointer that was smudged, and an error, if one occurred.
+func delayedSmudge(s *git.FilterProcessScanner, to io.Writer, from io.Reader, q *tq.TransferQueue, filename string, skip bool, filter *filepathfilter.Filter) (int64, bool, *lfs.Pointer, error) {
```

Comment:
Even though this is a long line, I can't think of any other function signatures that we wrap over multiple lines in LFS. For the sake of consistency, I'm going to leave this here. This long line has the added benefit of pointing out a smell that should be fixed.